scripts/net/run-sample-tests.sh

```diff
@@ -0,0 +1,262 @@
+#!/bin/sh
```

Comment:
commit message talks about echo_server, this however is not related to echo_server.

Reply:
Fixed.